dlt-database/entity/index.ts

```diff
@@ -0,0 +1,19 @@
+import { Account } from './Account'
+import { AccountCommunity } from './AccountCommunity'
+import { Community } from './Community'
+import { ConfirmedTransaction } from './ConfirmedTransaction'
+import { InvalidTransaction } from './InvalidTransaction'
+import { Migration } from './Migration'
+import { TransactionRecipe } from './TransactionRecipe'
+import { User } from './User'
+
+export const entities = [
+  AccountCommunity,
+  Account,
+  Community,
+  ConfirmedTransaction,
+  InvalidTransaction,
+  Migration,
+  TransactionRecipe,
+  User,
```

Comment:
only a hint:
we use in the database the naming conventions for tables in plural, means:
table name = Users, Communities,...
entity name = User, Community,...

Reply:
Yeah I know, because the reason is first I used cakephp for the community an there the standard is using this type of name conventions. As we switched over to NodeJs where the name convention is using singular we keep using the plural names for tables but not for entities. Therefore we need to use the @Column({ name: 'plural_table_name' }) to tell typeorm how the table is named, else it would simply use the Entity name directly.